rcl/test/rcl/test_graph.cpp

```diff
@@ -99,6 +100,10 @@ class CLASSNAME (TestGraphFixture, RMW_IMPLEMENTATION) : public ::testing::Test
     const char * name = "test_graph_node";
     ret = rcl_node_init(this->node_ptr, name, "", this->context_ptr, &node_options);
     ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+    if (rcl_logging_rosout_enabled() && node_options.enable_rosout) {
```

Comment:
`test_graph.cpp` doesn't use the `enable_rosout`. Is it necessary to add these code in `test_graph.cpp`?

Reply:
Currently not necessary, but for test stability i would like to keep this to check the node option as well.